api/datastore/test_data/input_24.json

```diff
@@ -1442,6 +1442,7 @@
       "Env": [],
       "Id": 2,
       "Name": "alpine",
+      "Logo": "https://portainer-io-assets.sfo2.digitaloceanspaces.com/logos/portainer.png",
```

Comment:
Don't edit this file, it's to test migrations from DBVersion 24.  This file is a representation of what a DBVersion 24 DB looks like.   Only the output file (the one ending in latest) should be changed.  You can copy the expected output file from the /tmp dir (in reverse order of failing) to the latest.   You'll see what I mean when you `run go test ./...`